### ISSUE TITLE
I want to know if you uesd the weight decay in pytorch?

### ISSUE THREAD

**Issue by solon37** [fsschneider/DeepOBS#38]
I am looking forward to hearing from you.

**Comment by fsschneider** [comment]
Whether the testproblems use weight decay (and if so for what part) is usually described in the comment of each testproblem. For example for the `3c3d` network on `cifar100`:

https://github.com/fsschneider/DeepOBS/blob/bfb1eb35e45b913f916dddf15165d23e0a40de00/deepobs/pytorch/testproblems/cifar100_3c3d.py#L21-L22

This is also listed in the documentation, e.g. [here ](https://deepobs.readthedocs.io/en/v1.2.0-beta0_a/api/pytorch/testproblems/mnist/mnist_mlp.html)for the `mlp` on `MNIST`.

Table 1 of the [DeepOBS paper](https://arxiv.org/pdf/1903.05499.pdf) has an overview table of the original testproblem and marked which ones use weight decay (WD).
